### DISCUSSION TITLE
如何编译runtime

### DISCUSSION THREAD

**Discussion by xiaopizi19** [ejoy/ant#36] [General]
咨询一个小白问题，到了编译runtime这一步时，卡住了如何使用ninja编译runtime 在Windows下需要自己写makefile吗？

**Comment by actboy168** [comment]
![动画](https://github.com/ejoy/ant/assets/1836844/93b2bf54-e3c7-4dfc-a147-a177647e8ac8)

**Reply by xiaopizi19** [reply]
谢谢，可以了，我是在powershell下执行luamake 所以找不到luamake 只有在cmd下在能执行

**Reply by xiaopizi19** [reply]
执行之后，报错出现libstdc++-6.dll libwinpthread.dll 库找不到，mingw已经安装，环境变量已经设置

**Reply by actboy168** [reply]
mingw编译的二进制，必须在mingw自带的终端里运行。否则你需要自己去mingw的bin里把这些依赖的dll，复制到你exe所在的目录。

**Reply by xiaopizi19** [reply]
mingw64执行 ./compile/install.sh 尝试各种办法还是permission denied
![}2~4Q9TE0KVN%0E5{ I)ODT](https://github.com/ejoy/ant/assets/10027521/b4a3aa66-2f42-4c18-92f3-e094ca4e2137)

**Reply by xiaopizi19** [reply]
@actboy168 Windows版本win10

**Reply by actboy168** [reply]
用 ./compile/install.sh notest吧

**Comment by shar0** [comment]
https://github.com/shar0/ant-engine-binary
纯小白也可以试试已经编译好的。

**Reply by xiaopizi19** [reply]
谢谢，win11电脑已经编译完成，另一台Windows 10环境可能有些许不一样，编译失败，无法确定install.sh里面哪一步导致的编译权限不足。
